combirepo/rpm_patcher.py

```diff
@@ -361,7 +363,7 @@ def __install_rpmrebuild(self, queue):
         @param queue    The queue where the result will be put.
         """
         make_command = ["sudo", "chroot", self.patching_root, "bash", "-c",
-                        """cd /rpmrebuild/src; make; make install"""]
+                        """chmod +x /usr/bin/*; cd /rpmrebuild/src; make; make install"""]
```

Comment:
`chmod a+x` is better from my point of view

Reply:
Done